src/routes/v2/authenticated/sites.ts

```diff
@@ -92,15 +92,10 @@ export class SitesRouter {
     { userWithSiteSessionData: UserWithSiteSessionData }
   > = async (req, res) => {
     const { userWithSiteSessionData } = res.locals
-    const possibleSiteUrl = await this.sitesService.getSiteUrl(
-      userWithSiteSessionData
-    )
-
-    // Check for error and throw
-    if (possibleSiteUrl instanceof BaseIsomerError) {
```

Comment:
Nit: Remove import for `BaseIsomerError`

Reply:
done!